templates/named.conf.options.erb

```diff
@@ -1,5 +1,5 @@
 options {
-	directory "/var/cache/bind";
+	directory "<%= scope.lookupvar('::dns::server::params::data_dir') %>";
```

Comment:
can you use `@data_dir` instead?

This more in-line with the style, and also allows the parameter to be easily overridden.